lib/puppet/indirector/hiera.rb

```diff
@@ -4,7 +4,7 @@
 class Puppet::Indirector::Hiera < Puppet::Indirector::Terminus
   def initialize(*args)
     if ! Puppet.features.hiera?
-      raise "Hiera terminus not supported without hiera library"
+      raise _("Hiera terminus not supported without hiera library")
```

Comment:
Maybe add a translator note that Hiera is an application name?

Also that library refers to a code library?